.gitignore

```diff
@@ -0,0 +1,27 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
+dist
+dist-ssr
+*.local
+
+# Editor directories and files
```

Comment:
Should put these in your [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) rather than add them to every repo you work in.